### DISCUSSION TITLE
Numbered Bullet List Restart

### DISCUSSION THREAD

**Discussion by arpit0515** [dolanmiu/docx#1966] [Q&A] [answered]
I am trying to create a document that has multiple paragraphs with numbered bullet points. I have my code that based on loop adds text runs and then I push it into new Paragraph. So the text runs become children of paragraph and then there is numbering property that I apply. I have set the reference and the level along with the children to be the text run. Unfortunately, for all the paragraphs, the numbering list keeps on incrementing for text runs (children of different paragraphs) even if they are part of separate paragraphs. How to reset the bullet list to be able to start from 1 again in new paragraph?

**Comment by dolanmiu** [comment, marked as answer]
Have you tried this:

https://github.com/dolanmiu/docx/blob/master/demo/68-numbering-instances-and-starting-number.ts
